index.html

```diff
@@ -53,5 +53,257 @@ <h2 class="section-header-subtitle">I’m a Software Developer</h2>
+            <li class="btn-lang ">Ruby</li>
+        </ul>
+<div class="button-see-proj">
+  <button class="see-project">
+      See Project
+  </button>
+</div>
+      </div>
+      </section>
+  </section>
+
+
+  <section class="background-image">
+    <section class="image-content">
+        <div class="image-cont-holder">
+        <h1 class="image-h1">
+            Profesional Art Printing Data
+        </h1>
+        <p class="para">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+        </p>
+
+        <ul class="languages-2">
+          <li class="btn-lang ">HTML</li>
+          <li class="btn-lang ">Bootstrap</li>
+          <li class="btn-lang ">Ruby</li>
+      </ul>
+<div class="button-see-proj">
+<button class="see-project">
+    See Project
+</button>
+</div>
+    </div>
+    </section>
+</section>
+
+<section class="background-image">
+  <section class="image-content">
+      <div class="image-cont-holder">
+      <h1 class="image-h1">
+          Profesional Art Printing Data
+      </h1>
+      <p class="para">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+      </p>
+
+      <ul class="languages-2">
+        <li class="btn-lang ">HTML</li>
+        <li class="btn-lang ">Bootstrap</li>
+        <li class="btn-lang ">Ruby</li>
+    </ul>
+<div class="button-see-proj">
+<button class="see-project">
+  See Project
+</button>
+</div>
+  </div>
+  </section>
+</section>
+
+<section class="background-image">
+  <section class="image-content">
+      <div class="image-cont-holder">
+      <h1 class="image-h1">
+          Profesional Art Printing Data
+      </h1>
+      <p class="para">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+      </p>
+
+      <ul class="languages-2">
+          <li class="btn-lang ">HTML</li>
+          <li class="btn-lang ">Bootstrap</li>
+          <li class="btn-lang ">Ruby</li>
+      </ul>
+<div class="button-see-proj">
+<button class="see-project">
+  See Project
+</button>
+</div>
+  </div>
+  </section>
+</section>
+
+<section class="background-image">
+  <section class="image-content">
+      <div class="image-cont-holder">
+      <h1 class="image-h1">
+          Profesional Art Printing Data
+      </h1>
+      <p class="para">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+      </p>
+
+      <ul class="languages-2">
+        <li class="btn-lang ">HTML</li>
+        <li class="btn-lang ">Bootstrap</li>
+        <li class="btn-lang ">Ruby</li>
+    </ul>
+<div class="button-see-proj">
+<button class="see-project">
+  See Project
+</button>
+</div>
+  </div>
+  </section>
+</section>
+
+
+</section>
+</section>
+<section class="about-me">
+  <section class="about-me-page">
```

Comment:
- [ ] Well done on adding and styling about me section but kindly stick as much as you can to [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471).

Here you have a screenshot from your project:

![image](https://user-images.githubusercontent.com/74506933/159077742-11c28990-48ce-47fe-b8a2-b816429c5ce8.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/159077784-254610a2-d3b0-49b3-be8b-34f9522a1cf9.png)

Reply:
@Donard97  I think what I need to address here is the width. Kindly respond fast because I am seriously behind schedule.